vespajlib/pom.xml

```diff
@@ -38,6 +38,11 @@
       <artifactId>guava</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
Note: Several modules uses `vespajlib` in compile scope - some bundles may require additional Jackson packages.

Reply:
Good point, I have removed the dependency and use Slime instead.

I have discussed the Slime changes with @havardpe .